.travis.yml

```diff
@@ -32,7 +34,8 @@ after_success:
 
 env:
   global:
-  - TRAVIS_CARGO_NIGHTLY_FEATURE=nightly
+  - TRAVIS_CARGO_NIGHTLY_FEATURE=nightly,deadlock_detection
```

Comment:
TRAVIS_CARGO_NIGHTLY_FEATURE should be left as it was before. The `deadlock_detection` feature should be explicitly tested separately.